test/index.html

```diff
@@ -11,40 +11,45 @@
     <h1>Checka11y.css - test.html</h1>
     <p>The intention of this file is to test out the new a11y checks you are adding, so this web page should try to not comply with a11y as much as possible.</p>
     <hr />
+    <section>
+      <h2>HTML: html tag should have a lang attribute</h2>
+      <p>See top of page.</p>
+    </section>
     <section>
       <h2>Images: All images require an alt attribute, whether it has a value or is null</h2>
       <img src="./assets/a11y.png" />
     </section>
     <section>
       <h2>iframe: iframe has title attribute</h2>
-      <iframe src="https://jackdomleo.dev" height="500" width="500"></iframe>
+      <iframe src="https://checka11y.jackdomleo.dev" height="500" width="500"></iframe>
     </section>
     <section>
       <h2>Lists: All children of &lt;ol&gt; must only be &lt;li&gt;</h2>
       <ol>
-        <div>Hello, I'm a div inside a &lt;ol&gt;</div>
+        <li>Hello, I'm a &lt;li&gt; inside a &lt;ol&gt;</li>
+        <div>Hello, I'm a &lt;div&gt; inside a &lt;ol&gt;</div>
```

Comment:
I wonder if there's a way where we can remove all of these ugly html symbol codes ’\&lt;’ and replace them with something more readable.

Reply:
I'll do some digging. These are common if you want to display `<` as text rather than render it as HTML. But I can have a look to see if we can still have `<`.

Reply:
Yeah I get that, but as we're going to be dealing with tags a lot this might be annoying. Not the end of the world of we do need to keep them though.

Reply:
I think we'll keep them for now. I find seeing `<p>` rendered more clarifying rather than `p` or `p tag/element`. If this means writing `&lt;p&gt;`, then I'm ok with that for now. I can't think of an alternative solution right now. 🤔